guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -76,14 +77,89 @@ ifdef::katello[]
 # foreman-rake katello:upgrade_check
 ----
 endif::[]
+ifdef::foreman-deb[]
+. Update repositories in `/etc/apt/sources.list.d/foreman.list` to consume content for {Project} {ProjectVersion}:
++
+[options="nowrap" subs="attributes"]
+----
+deb http://deb.theforeman.org/ _My_Distribution_Code_Name_ {ProjectVersion}
+deb http://deb.theforeman.org/ plugins {ProjectVersion}
+----
++
+// List of supported OS for foreman-deb has to match "guides/common/modules/proc_configuring-repositories-foreman-deb.adoc"
+Replace _My_Distribution_Code_Name_ with the proper distribution code name based on the operating system of your {ProjectServer}:
++
+* `bullseye` for Debian 11
+* `jammy` for Ubuntu 22.04
+* `focal` for Ubuntu 20.04
+. Update to {Project} {ProjectVersion}:
++
+[options="nowrap" subs="attributes"]
+----
+# apt-get update
+# apt-get --only-upgrade install ruby\* foreman\*
+----
+. Migrate the database:
```

Comment:
I would prefer to take this opportunity and drop all non-installer setups from the docs.
Then the upgrade becomes:
1. stop services
2. apt update/apt upgrade
3. foreman-installer

Reply:
(we do so in EL-docs already.